### PR TITLE
Run a 'terraform plan' in Travis

### PR DESCRIPTION
**Pull request by alexwlchan**
This patch adds a test step in Travis to run `terraform plan` before merging to master. This will allows us to spot if somebody is trying to commit bad config.

Ahem.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    sudo mv terraform /usr/bin/terraform;
+    cd $repo_dir
+  - pip install --upgrade --user awscli
+
+script:
+  - cd terraform; ./plan.sh
+
+env:
+  matrix:
+    secure: Hxp/gtLCE0cx2UMv31qPKo3GdnLxANmjS/HzHBzhCs5zcpMHnuRPCPdAW5RSbPRvEKHBiZ6ULp2EaS2PPCdJQnmwueDoO7Ou/RWvlbzbPElz2EY0o0sNhFVVRB28uz06asIStz+UoVLN+G3grOzKsuUsoU0PE6FWkpH4MvPv83RXZ18uZuiaJJIVmaUiVetfktL+rFodRLiY2cDFLGjfswEL/Dgc9rZOFN6r7Fo/svJIgHklFg3t/rDMSB4t+zuOA9oxxrGBVz3JiWK30InbrKXhZKLw2UdEZsEI6ao0xNN++NkE7L3KNo1sF7TeKKlehJvWjXMl9hssdFdO/LCDDst0ipmfnCmSuERdvndbiT089Yx2KEH0LDgtLBtNUKy76CSc3HL2aiQZpbvOmeqiSLL/sasQP/DftFn9SGgWSXdiDYkDyBbKx5oAFgI9Og1aIEHUpuyLBGWJQx/rPHD1rOnKLjKgv0u6mE3MtV5AORJU8Yb4H67weJnq0Ufw+PQs87zWPUvO7kBsoq62VEhAf3KpPN4oz1DfijnzCJJSNxWA3N4rIpcP0LevanKR/xL/qHn85A5URJvdv6z8GQQgnbLesOrVwqlMJVYnZA0z8oIKimhTvndWNI8DXX5JD1jDzC3yHvdMiFTwx+tgYBqwF/r1KfEoIU+63o75RJgxY2I=
```
Comment (alexwlchan) [inline .travis.yml]: This is the encrypted access key/secret ID for a new Travis user in AWS, which has read access to S3 and full access to ECR. See: https://docs.travis-ci.com/user/environment-variables/#Defining-encrypted-variables-in-.travis.yml

Comment (alexwlchan) [inline .travis.yml]: Okay, and read-only access to a few other resources, or terraform gets confused.

**Comment by alexwlchan** [comment]
Anyway, this is ready for review now. Ping @alicefuzier and @kenoir.

**Comment by kenoir** [comment]
Looks good.

**Pull request merged**